lib/index.js

```diff
@@ -249,3 +249,110 @@ internals.uniqueObjects = (arr) => {
 
     return uniqueObjects;
 };
+
+exports.Model = class SchwiftyModel extends Objection.Model {
```

Comment:
No need to re-add the entire model into lib/index.js.  You can go ahead and make the changes in lib/model.js.